opendebates/static/less/base.less

```diff
@@ -168,12 +168,54 @@ body > .container {
   }
 
   > .warning {
-    font-family: Knockout;
-    font-weight: bold;
     text-align: center;
-    background-color: red;
+    background-color: @brand-tertiary;
     color: white;
     padding: 10px;
+    .full({
+      min-height: 250px;
+      padding: 10px 280px;
+      position: relative;
+    });
+
+    &::before {
+      .full({
+        content: "";
+        background-image: url(/static/images/megaphone-left.png);
+        left: 15px;
+        bottom: 0;
+        height: 130px;
+        width: 176px;
+        position: absolute;
+      });
+    }
+    &::after {
+      .full({
+        content: "";
+        background-image: url(/static/images/megaphone-right.png);
+        right: 15px;
+        bottom: 0;
+        height: 130px;
+        width: 176px;
+        position: absolute;
+      });
+    }
```

Comment:
Yeah, I figured I might as well apply these to the upgrade banner too -- though I didn't actually think to check how it looks on old browsers.  Do you think that's risky?  Would these styles be likely to mangle the upgrade banner on the browsers that need to see it?